cfgov/apache/conf.d/deleted.conf.sample

```diff
@@ -0,0 +1,7 @@
+# Serve deleted content JSON archives at /__deleted
+PassEnv ARCHIVE_PATH
+LoadModule autoindex_module modules/mod_autoindex.so
+Alias /__deleted/ "${ARCHIVE_PATH}"
+<Directory "${ARCHIVE_PATH}">
```

Comment:
```suggestion
PassEnv WAGTAIL_DELETION_ARCHIVE_PATH
LoadModule autoindex_module modules/mod_autoindex.so
Alias /__deleted/ "${WAGTAIL_DELETION_ARCHIVE_PATH}"
<Directory "${WAGTAIL_DELETION_ARCHIVE_PATH}">
```

Reply:
Oops, I caught that too! Fixed in the latest push.